test/torch/model/test_estimators.py

```diff
@@ -123,17 +131,35 @@ def test_estimator_constant_dataset(
     estimator_constructor, use_validation_data: bool
 ):
     constant = get_dataset("constant")
-
     estimator = estimator_constructor(constant)
 
+    if isinstance(estimator, ITransformerEstimator):
```

Comment:
This will not scale, @kashif how about copying the test (it’s not so long anyway) for the multi variate setting? `test_estimator_multivariate_constant_dataset`